src/mapillary_client.py

```diff
@@ -0,0 +1,71 @@
+from os import getenv, getcwd
+import logging
+from multiprocessing import Pool
+from pathlib import Path
+from mapillary import interface as mly
+from requests import get, RequestException
+
+
+class MapillaryClient:
+    def __init__(self, access_token: str, log_level: int = logging.INFO, basepath: str = getcwd()) -> None:
+        self.access_token = access_token
+        mly.set_access_token(access_token)
+        self.log = logging.getLogger(self.__class__.__name__)
+        self.log.addHandler(logging.StreamHandler())
+        self.log.setLevel(log_level)
```

Comment:
Not a strongly held view, but I personally like loguru a lot as a logging library. It has a lot of nice defaults and lets you avoid needing to set up boilerplate. 

https://github.com/Delgan/loguru

Also, I've more commonly seen loggers initialized at the module level, and the level controlled with global log level settings. Attaching the logger to the client class feels unnecessary to me.

Reply:
I am more familiar with Java's Slf4J, which attaches the logger at the class level to provide the context class and method names at each log. Evidently, this did not happen with the `self.__class__.__name__`, so I am open to suggestions.